modules/core/src/minmax.dispatch.cpp

```diff
@@ -0,0 +1,498 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+
+
+#include "precomp.hpp"
+#include "opencl_kernels_core.hpp"
+#include "opencv2/core/openvx/ovx_defs.hpp"
+#include "stat.hpp"
+#include "opencv2/core/detail/dispatch_helper.impl.hpp"
+#include <algorithm>
+
+#include "minmax.simd.hpp"
+#include "minmax.simd_declarations.hpp" // defines CV_CPU_DISPATCH_MODES_ALL=AVX2,...,BASELINE based on CMakeLists.txt content
+
+namespace cv {
+
+static MinMaxIdxFunc getMinMaxIdxFunc(int depth)
+{
+    CV_INSTRUMENT_REGION();
+    CV_CPU_DISPATCH(getMinMaxIdxFunc, (depth),
+                    CV_CPU_DISPATCH_MODES_ALL);
+}
+
+static void ofs2idx(const Mat& a, size_t ofs, int* idx)
+{
+    int i, d = a.dims;
+    if( ofs > 0 )
+    {
+        ofs--;
+        for( i = d-1; i >= 0; i-- )
+        {
+            int sz = a.size[i];
+            idx[i] = (int)(ofs % sz);
+            ofs /= sz;
+        }
+    }
+    else
+    {
+        for( i = d-1; i >= 0; i-- )
+            idx[i] = -1;
+    }
+}
+
+#ifdef HAVE_OPENCL
```

Comment:
1. Git history of these changes has been lost in this PR (missing explicit git rename/copy).
2. This guarantee 100% merge conflicts in the future against 4.x branch (yep, you don't care about "merge 4.x" requests, even no reviewing activity).
3. Just take a look on the history of other .dispatch.cpp files (...)